docs/misc/faq.md

```diff
@@ -101,7 +101,7 @@ with several powerful functionalities:
  where thousands of people have uploaded useful containers: anything from Redis,
```

Comment:
While we're at it, perhaps we should change the above link to https://registry.hub.docker.com. The current link redirects to the signup page, which may not be what we want here (we want to show what images are available, I think?)